polaris-core/src/main/resources/schemas/policies/system/orphan-file-removal/2025-02-03.json

```diff
@@ -0,0 +1,50 @@
+{
+  "license": "Licensed under the Apache License, Version 2.0 (http://www.apache.org/licenses/LICENSE-2.0)",
+  "$id": "https://polaris.apache.org/schemas/policies/system/orphan-file-removal/2025-02-03.json",
+  "title": "Orphan File Removal Policy",
+  "description": "Inheritable Polaris policy schema for Iceberg table orphan file removal",
+  "type": "object",
+  "properties": {
+    "version": {
+      "type": "string",
+      "const": "2025-02-03",
+      "description": "Schema version"
+    },
+    "enable": {
+      "type": "boolean",
+      "description": "Enable or disable orphan file removal."
+    },
+    "max_orphan_file_age_in_days": {
```

Comment:
@flyrain this also struggles me a bit, remove orphan policy can be even expressed in more than in just file age. We don't we opt for the `config` key similar to the other policies?

Reply:
> remove orphan policy can be even expressed in more than in just file age.

Can you name them? We can put them into schema if they are commonly used, otherwise, the config map would be the best place to be.

Reply:
I'd vote for the config map

Reply:
If no extra field is suggested, we could keep it as is.